components/financial_assistance/app/domain/financial_assistance/operations/applications/medicaid_gateway/add_eligibility_determination.rb

```diff
@@ -106,6 +109,19 @@ def update_applicants(elig_d, thh_entity)
               build_member_determinations(applicant, member_determinations) if member_determinations
               applicant.save
             end
+
+            notify_totally_ineligible(application) if totally_ineligible_members.include?(true) && FinancialAssistanceRegistry.feature_enabled?(:totally_ineligible_notice)
```

Comment:
Do we want to notify when at-least one member is ineligible or if all the members are ineligible ?

Reply:
at-least one